formidable/app.py

```diff
@@ -25,9 +20,3 @@ def ready(self):
         """
         from .views import check_callback_configuration
         check_callback_configuration()
-
-        # Deprecation warning for django 1.8 and 1.9
-        if StrictVersion(django.get_version()) < StrictVersion("1.10"):
-            warnings.warn("Support for Django 1.8 and 1.9 is about to be "
```

Comment:
RIP deprecation warning 20 April 2018 - 20 April 2018 never forgotten

Reply:
nope... we don't know yet when the 2.0.0 would be released. And we could have people who use a 1.x version until they finally switch to 1.10 for quite some time.
don't put too much affect... it's just software after all ;o)